src/app/pages/ParaTimesPage/TransactionRecipient/index.tsx

```diff
@@ -92,6 +95,21 @@ export const TransactionRecipient = () => {
               value={transactionForm.ethPrivateKey}
               showTip={t('openWallet.privateKey.showPrivateKey', 'Show private key')}
               hideTip={t('openWallet.privateKey.hidePrivateKey', 'Hide private key')}
+              suggestions={evmAccounts.map(acc => ({ label: acc.ethAddress, value: acc.ethPrivateKey }))}
+              onSuggestionSelect={event => {
+                // Add timeout because grommet reassigns value after onSuggestionSelect
+                // https://github.com/grommet/grommet/blob/7ad63c7/src/js/components/TextInput/TextInput.js#L279-L281
+                // But why is timeout is not needed in SendTransaction??
+                // https://github.com/oasisprotocol/oasis-wallet-web/blob/fa173f7/src/app/pages/AccountPage/Features/SendTransaction/index.tsx#L89-L91
+                setTimeout(() => {
+                  setTransactionForm({
+                    ...transactionForm,
+                    ethPrivateKey:
```

Comment:
Grommet's internals trigger `index.js:1 Warning: Failed prop type: Invalid prop `value` supplied to `TextInput`, expected one of type [string, number].` by assigning object `{label, value}` to `value`.

I don't see a way to prevent it. I don't understand why https://github.com/oasisprotocol/oasis-wallet-web/pull/1735/files#diff-e938730bd232b3bdc6fc39b9602c930d4eacd6a4dad6af7918459917aa085837R89-R92 doesn't trigger the same.

Happy to ignore

Reply:
value is an object not a string in when you check Form onChange
{
    "amount": "",
    "confirmTransfer": false,
    "confirmTransferToValidator": false,
    "confirmTransferToForeignAccount": false,
    "defaultFeeAmount": "",
    "ethPrivateKey": {
        "label": "...",
        "value": "..."
    },
    "feeAmount": "",
    "feeGas": "",
    "paraTime": "sapphire",
    "recipient": "",
    "type": "withdraw"
}
```

Reply:
kill 'global' Form onChange and add it to each input field?

```
            <TextInput
              onChange={event =>
                setTransactionForm({
                  ...transactionForm,
                  recipient: event.target.value,
                })
              }
```

(didn't test it)

Reply:
What do you think of keeping mostly as is, removing onSuggestionSelect, and:
```tsx
      <Form
        onChange={nextValue =>
          setTransactionForm({
            ...nextValue,
            ethPrivateKey:
              typeof nextValue.ethPrivateKey === 'object'
                ? (nextValue.ethPrivateKey as any).value // from suggestions
                : nextValue.ethPrivateKey,
          })
        }
```

Reply:
I am fine with whatever works